include/exec/cpu-defs.h

```diff
@@ -165,10 +167,16 @@ typedef struct CPUIOTLBEntry {
      */
     uintptr_t tagmem_read;
     /*
-     * ALL_ZERO_TAGBLK without either trapping or zero-ing is an invalid entry
-     * for tagmem_write and should cause a TLB miss if setting a tag to 1.
+     * There are two reasons to trap on writng a tag: first, if MMU protection
```

Comment:
```suggestion
     * There are two reasons to trap on writing a tag: first, if MMU protection
```
though the grammar of this is bad, you can't split across two sentences like this; either merge them or swap the colon for a full stop and add a verb to the new sentence.